collections/open-source-organizations/index.md

```diff
@@ -14,6 +14,7 @@ items:
  - IBM/ibm.github.io
  - Microsoft/microsoft.github.io
  - artsy/artsy.github.io
+ - OSGeo/OSGeo/OSGeo
```

Comment:
Sorry! This needs to be `OSGeo/OSGeo` which is the user or organisation, a `/` and then the repository (like those above)